lib/grit_adapter/git_layer_grit.rb

```diff
@@ -166,6 +166,12 @@ def rev_list(options, *refs)
       rescue Grit::GitRuby::Repository::NoSuchShaFound
         raise Gollum::Git::NoSuchShaFound
       end
+
+      def revert(path, sha1, sha2, ref)
+        patch = path ?
+          repo.diff(sha2, sha1, path).first.diff : repo.diff(sha2, sha1).map { |d| d.diff }.join("\n")
+        apply_patch(ref, patch)
```

Comment:
No, part of the point was to get rid of `apply_patch` as a public method, so you're right -- might as well get rid of it alltogether.